doc/source/docker/tls.rst

```diff
@@ -0,0 +1,148 @@
+  `Enable SSL connections <https://flower.ai/docs/framework/how-to-enable-ssl-connections.html#certificates>`__
+  page contains a section that will guide you through the process.
+
+
+Because Flower containers, by default, run with a non-root user ``app``, the mounted files and
+directories must have the proper permissions for the user ID ``49999``.
+
+For example, to change the user ID of all files in the ``certificates/`` directory, you can run
+``sudo chown -R 49999:49999 certificates/*``.
+
+If you later want to delete the directory, you can change the user ID back to the current user
+ID by running ``sudo chown -R $USER:$(id -gn) state``.
+
+SuperLink
+---------
+
+Assuming all files we need are in the local ``certificates`` directory, we can use the flag
+``--volume`` to mount the local directory into the ``/app/certificates/`` directory of the container:
+
+
+.. code-block:: bash
+   :substitutions:
+
+   $ docker run --rm \
+        --volume ./certificates/:/app/certificates/:ro \
+        flwr/superlink:|current_flwr_version| \
+        --ssl-ca-certfile certificates/ca.crt \
+        --ssl-certfile certificates/server.pem \
+        --ssl-keyfile certificates/server.key
+
+.. dropdown:: Understanding the command
+
+   * ``docker run``: This tells Docker to run a container from an image.
+   * ``--rm``: Remove the container once it is stopped or the command exits.
+   * | ``--volume ./certificates/:/app/certificates/:ro``: Mount the ``certificates`` directory in
+     | the current working directory of the host machine as a read-only volume at the
+     | ``/app/certificates`` directory inside the container.
+     |
+     | This allows the container to access the TLS certificates that are stored in the certificates
+     | directory.
+   * | :substitution-code:`flwr/superlink:|current_flwr_version|`: The name of the image to be run and the specific
+     | tag of the image. The tag :substitution-code:`|current_flwr_version|` represents a specific version of the image.
+   * | ``--ssl-ca-certfile certificates/ca.crt``: Specify the location of the CA certificate file
+     | inside the container.
+     |
+     | The ``certificates/ca.crt`` file is a certificate that is used to verify the identity of the
+     | SuperLink.
+   * | ``--ssl-certfile certificates/server.pem``: Specify the location of the SuperLink's
+     | TLS certificate file inside the container.
+     |
+     | The ``certificates/server.pem`` file is used to identify the SuperLink and to encrypt the
+     | data that is transmitted over the network.
+   * | ``--ssl-keyfile certificates/server.key``: Specify the location of the SuperLink's
+     | TLS private key file inside the container.
+     |
+     | The ``certificates/server.key`` file is used to decrypt the data that is transmitted over
+     | the network.
+
+SuperNode
+---------
+
+Assuming the certificate already exists locally, we can use the flag ``--volume`` to mount the local
```

Comment:
```suggestion
Assuming that the ``ca.crt`` certificate already exists locally, we can use the flag ``--volume`` to mount the local
```

Should we explain that the `ca.crt` can be copied over after the self-signed generated certificates step, if it doesn't exist on the SuperNode?

Reply:
Yes, I think we can add a note here.